docs/proposals/2020-08-12-Collection_Query_Backfill.md

```diff
@@ -0,0 +1,69 @@
+# Collection Query Backfill 
+
+## Problem
+
+Backfilling Collections with query results is a complex process that requires implementation across multiple systems.
+This document proposes a first solution to achieve the near-term deliverable.
+
+## Proposal
+
+In order to limit scope on both the rendering and publishing sides, we only allow specifying a single section or tag. The Collections API would be updated to support this via the API and presumably UI as well.
+
+### Example
+
+We would update the ANS spec to allow specifying a single website or tag:
+
+```JSON
+{
+  "type": "collection",
+  "dynamic_items": {
+    "type": "sections"
```

Comment:
You say single section but the type is "sections" and it uses an array

Reply:
explained the use of plural and array